numpy/linalg/umath_linalg.c.src

```diff
@@ -706,6 +706,11 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
+#ifndef min
+#define min(x,y) ((x<y)?x:y)
```

Comment:
@charris: How does the latest commit look to address this? If it's alright, I'll squash it over the previous commit